src/main/scala/com/datastax/gatling/plugin/request/GraphRequestAction.scala

```diff
@@ -69,7 +71,11 @@ class GraphRequestAction(val name: String,
       ThroughputVerifier.checkForGatlingOverloading(session, gatlingTimingSource)
       GatlingResponseTime.startedByGatling(session, gatlingTimingSource)
     }
-    val stmt = dseAttributes.statement.buildFromSession(session)
+
+    // Attempt to generate a graph statement from our parameters, propagating any uncaught exceptions after
+    // continuing the chain with a failed session
+    val stmt: Validation[GraphStatement] =
+      dseAttributes.statement.buildFromSession(session)
```

Comment:
I think the comment lines are not accurate anymore here, could you restore the original line?

Reply:
Good catch, I'll delete that comment